source/framework/vx_objarray.c

```diff
@@ -453,34 +489,31 @@ static vx_status ownDestructObjArray(vx_reference ref)
     vx_status status = (vx_status)VX_SUCCESS;
     vx_object_array objarr = NULL;
 
-    if(ref->type == (vx_enum)VX_TYPE_OBJECT_ARRAY)
+    /* status set to NULL due to preceding type check */
```

Comment:
After removing type check, this comment needs to be updated:
@lcweaver , should it be:
1. updated to "status check set to NULL due to guaranteed type in internal function" or
2. removed and NULL changed to non-NULL.

Reply:
done